lib/spack/spack/test/cmd/list.py

```diff
@@ -84,3 +84,9 @@ def test_list_update(tmpdir, mock_packages):
     assert update_file.exists()
     with update_file.open() as f:
         assert f.read() == "empty\n"
+
+
+def test_tag1(parser, specs):
```

Comment:
All you should need here are the mock packages:

```suggestion
def test_tag1(mock_packages):
```